duckling/language.py

```diff
@@ -58,3 +58,13 @@ def convert_to_duckling_language_id(cls, lang):
         else:
             raise ValueError("Unsupported language '{}'. Supported languages: {}".format(
                 lang, ", ".join(cls.SUPPORTED_LANGUAGES)))
+
+    @classmethod
+    def convert_to_iso(cls, lang):
```

Comment:
The reasoning here was that the `load` function of duckling supports iso codes only. But, yes, that could definitely be cleaned. I'll fix that.